src/styles/_catalog.scss

```diff
@@ -20,3 +21,21 @@
     margin-right: 48px;
   }
 }
+
+@media (max-width: 1023.9px) {
+  .catalog {
+    max-width: 940px;
+  }
+}
+
+@media (max-width: 767.9px) {
+  .catalog {
+    max-width: 690px;
+  }
+}
+
+@media (max-width: 487.9px) {
+  .catalog {
+    max-width: 420px;
+  }
+}
```

Comment:
add it inside .catalog selector then you leave just `@media { styles }`
Also you shouldn't change max-width
The point is to change grid-settings so use 
`grid-template-columns: repeat({number}, 200px);`